crates/astria-conductor/src/config.rs

```diff
@@ -61,6 +61,12 @@ pub struct Config {
     /// The number of requests per second that will be sent to Sequencer.
     pub sequencer_requests_per_second: u32,
 
+    /// The chain ID of the sequencer network the conductor should be communiacting with.
+    pub sequencer_chain_id: String,
```

Comment:
bikeshedding: i wonder if calling this `expected_sequencer_chain_id` would be more descriptive.

(and same for the one below)